src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesConfiguration.cs

```diff
@@ -6,7 +6,7 @@ namespace Raven.Client.Documents.Operations.TimeSeries
 {
     public class TimeSeriesConfiguration : IDynamicJson
     {
-        internal const char TimeSeriesRollupSeparator = '@';
+        internal const char TimeSeriesRollupSeparator = '_';
```

Comment:
It means that we cannot use that char in TS name. The `_` is more popular than `@` I think?

Reply:
Where is this actually used?

Reply:
reverted it back to '@'

(The reason for the change was because we originally thought that the user will pass the roll-up names in query, and this '@' caused QueryScanner to not recognize these names as 'FieldExpression'.. We decided to go a different way so this doesn't matter anymore..)